### PR TITLE
[Fix-10652] fix docker-compose.yml

### LINKED ISSUE #10652: [Bug] [api] docker compose does not expose port 25333
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

docker compose does not expose port 25333 so that the python api cannot be called

### What you expected to happen

python api be called

### How to reproduce

use docker compose

### Anything else

_No response_

### Version

3.0.0-beta-1

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by ly1ex**
## Purpose of the pull request

This pull request fix #10652.

## Brief change log

- *Expose 25333 port to api server in docker-compose.yml*

## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@lyleshaw  Good catch! but could you also export the port 24333 to https://github.com/apache/dolphinscheduler/blob/75581b7543cfb2560090d349f9879a3d6fda142c/deploy/docker/docker-stack.yml#L52
and https://github.com/apache/dolphinscheduler/blob/b1885c7a26a5404ce42d8fb0f2d8533d9a7c83c1/deploy/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml#L63-L64
and 
https://github.com/apache/dolphinscheduler/blob/67cc260d52c1f2e5aa7db76aa8621cdd0f8c4ee0/deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml#L36-L42

**Comment by ly1ex** [comment]
> @lyleshaw Good catch! but could you also export the port 24333 to
> 
> https://github.com/apache/dolphinscheduler/blob/75581b7543cfb2560090d349f9879a3d6fda142c/deploy/docker/docker-stack.yml#L52
> 
> 
> and
> https://github.com/apache/dolphinscheduler/blob/b1885c7a26a5404ce42d8fb0f2d8533d9a7c83c1/deploy/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml#L63-L64
> 
> 
> and
> https://github.com/apache/dolphinscheduler/blob/67cc260d52c1f2e5aa7db76aa8621cdd0f8c4ee0/deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml#L36-L42

All completed
BTW, I think you want to say port 25333 instead of 24333
ping @zhongjiajie

**Comment by zhongjiajie** [comment]
> > @lyleshaw Good catch! but could you also export the port 24333 to
> > https://github.com/apache/dolphinscheduler/blob/75581b7543cfb2560090d349f9879a3d6fda142c/deploy/docker/docker-stack.yml#L52
> > 
> > and
> > https://github.com/apache/dolphinscheduler/blob/b1885c7a26a5404ce42d8fb0f2d8533d9a7c83c1/deploy/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml#L63-L64
> > 
> > and
> > https://github.com/apache/dolphinscheduler/blob/67cc260d52c1f2e5aa7db76aa8621cdd0f8c4ee0/deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml#L36-L42
> 
> All completed BTW, I think you want to say port 25333 instead of 24333 ping @zhongjiajie

Yeah, you're right , it is 25333 instead of 24333

File: deploy/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml [hunk 1]
```diff
@@ -62,6 +62,8 @@ spec:
           ports:
             - containerPort: 12345
               name: "api-port"
+            - containerPort: 25333
+                name: "python-api-port"
```
Comment (zhongjiajie) [inline deploy/kubernetes/dolphinscheduler/templates/deployment-dolphinscheduler-api.yaml]:
```suggestion
              name: "python-api-port"
```

**Review by zhongjiajie — DISMISSED** [review]

**Review by zhongjiajie — DISMISSED** [review]
LGTM, could you double check the chart setting is correct or not here? we want to export one more port here @kezhenxu94 @caishunfeng

File: deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml [hunk 2]
```diff
@@ -40,6 +40,13 @@ spec:
       {{- end }}
       protocol: TCP
       name: api-port
+    - port: 25333
+      targetPort: python-api-port
+      { { - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort } }
```
Comment (kezhenxu94) [inline deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml]: `{ {` is wrong in Helm Chart syntax, please remove the space between the 2 `{`

**Review by kezhenxu94 — CHANGES_REQUESTED** [review]

File: deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml (hunk 2 above)
Comment (ly1ex) [reply deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml]: > `{ {` is wrong in Helm Chart syntax, please remove the space between the 2 `{`

Done

File: deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml [hunk 3]
```diff
@@ -42,9 +42,9 @@ spec:
       name: api-port
     - port: 25333
       targetPort: python-api-port
-      { { - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort } }
-      nodePort: { { .Values.api.service.nodePort } }
-      { { - end } }
+      {{ - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort }}
```
Comment (kezhenxu94) [inline deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml]: Sorry I missed this, the space between `{{` and `-` should be removed too

Comment (ly1ex) [reply deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml]: > Sorry I missed this, the space between `{{` and `-` should be removed too

Done~

**Comment by lyleshaw** [comment]
It looks like E2E's FileManage is failing, maybe I can ask you to help me track down why it's failing 😵‍💫@zhongjiajie @kezhenxu94

**Review by kezhenxu94 — APPROVED** [review]

**Pull request merged**